### DISCUSSION TITLE
Error message pops up when running eKuiper manager with eKuiper as rules-engine for EdgeX

### DISCUSSION THREAD

**Discussion by hoangphuc1723** [lf-edge/ekuiper#3201] [Q&A]
I'm using the image lfedge/ekuiper:1.14.1-alpine of eKuiper as the rules-engine for EdgeX and the image emqx/ekuiper-manager:latest of eKuiper manager to provide the UI. The configuration and run was smooth, and I can connect to eKuiper as well as create streams, create rules, update rules, etc., but whenever I point my browser back to http://localhost:9082 and not logged in yet, an error message prompts "Invalid token: token contains an invalid number of segments". I wonder if there's a way to fix the error away, thank you so much in advance! Attached is eKuiper manager interface with the mentioned error message for context.
![image](https://github.com/user-attachments/assets/5a0eb466-6516-4f09-a599-144113a88d25)

**Comment by ngjaying** [comment]
Hi @hoangphuc1723. I am sorry manager has sun-set and no resource to maintain it now. Maybe you would like to take a look at https://www.emqx.com/en/products/neuronex, which integrates eKuiper and provide a web dashboard to manage.

**Comment by ankur-paan** [comment]
We are building an open source eKuiper manager at https://github.com/ankur-paan/ekuiper-manager. See if that helps. Also we are maintaining a live playground where you can interact with your own ekuiper instances using a browser at https://ankur-paan.github.io/ekuiper-manager/
